app/controllers/matchings_controller.rb

```diff
@@ -54,6 +58,111 @@ def destroy
     end
   end
 
+  def category
+    return "person_to_time"
+  end
+
+  def global_settings
+    return {"minutes": @project.duration}
+  end
+
+  def people
+    all_participants_emails = Participant.where(project_id: @project.id).pluck(:email)
```

Comment:
Rails tip for everytime you do this: can just do `@project.participants`, thanks to Rails Models Associations, where in the model Participant.rb we put `belongs_to project` and in Project.rb we put `has_many participants`

Same applies for `@project.project_times` which does `ProjectTime.where(project_id: @project.id)`
Isn't life so much easier? Thank you Rails ❤

Reply:
I've started making changes so expect a push in a few minutes